MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -552,4 +559,67 @@ - (void)selectScope:(id)sender
     [self.navigationController pushViewController:[MSALTestAppScopesViewController sharedController] animated:YES];
 }
 
+- (void)runStressTest:(id)sender
+{
+    (void)sender;
+    
+    void (^logHandler)(NSString *logLine) = ^(NSString *logLine) {
```

Comment:
This strikes me as a questionable use of asynchronous callbacks. I think I'd rather see notifications or almost anything else get used instead of pushing a block down the stack like that.